tests/README.md

```diff
@@ -0,0 +1,89 @@
+
+Currently we use logs parsing in order to assert our tests but that will be changed to using the node's API.
+
+####Packaging
+
+Spacemesh code is being run on containers, miner container is defined in `DockerFile` file.
+It includes a basic OS, environment variables, requirements (golang) and the compiled SM code.
+When running a make command (locally or using the CI) another container is being created, this container is running the 
+automation code and is defined under `DockerFileTests` file. It includes automation requirements (python, pytest, 
+py packages, etc) and the automation code itself to be run on this pod.
+
+###Kubernetes Files
+   
+K8s files are being used to deploy the networks' nodes (bootstrap node, client nodes, poet and curl).
+Those files are located under `k8s` folder.
+Important thing to know is that during the test before appliance these files are modified, it's being concatenated with
+the k8s configuration in the test config file and changes that are made in the code itself.
+You can see the changes being made in conftest.py under the functionality that deploys these files (setup bootstrap and
+clients for example).
+
+####SM-clients & Bootstrap
+When deploying SM-client the bootstrap node will be deployed first, next all other clients (miners) will be deployed with the bootstrap as their `entry point` to the network.
+Bootstrap node provides other nodes information about active nodes in the network.
+After deployment, bootstrap node will function as a normal miner. 
+After being deployed and part of the network, every client can perform as a bootstrap node for a new client that wants to join the network.
+The number of replicas for each deployment (bootsrap or miners) can be found in the tests config file.
+
+###Test Configurations
+
+Each test has it's own configuration file.
+This file is located at the same folder as the test and is named 'config.yaml' (except for late nodes test where it's
+named `delayed_config.yaml`).
+`config.yaml` consists of test configurations, k8s configurations and SM-client arguments which are being passed to the
+node on activation.
+The k8s configuration part of the config file will be concatenated to the k8s object files that are used by the test.
+
+One important variable in the config files is the 'genesis_delta', this is the representation in seconds of the time
+since deploying bootstrap node until the miners start mining, which means that this time should be sufficient for all
+nodes to be ready.
+If not ready by this time the test will fail.
+
+####Running Automation Locally
+
+Notice: Before running automation you need to have permissions to our GCP account management.
+There is a document named 'instructions' in 1password with detailed instructions on how to setup all requirements for 
+automation. 
+
+There are two options for running automation locally:
+
+* running pytest
+python3 -m pytest -s -v path/to/your_test.py --tc-file=path/to/your_test_config.py --tc-format=yaml
```

Comment:
````suggestion
```
python3 -m pytest -s -v path/to/your_test.py --tc-file=path/to/your_test_config.py --tc-format=yaml
```
````

Reply:
fixed